td/tracking/templates/tracking/event_form.html

```diff
@@ -37,6 +37,11 @@ <h2 class="panel-title">New Translation Workshop/Event</h2>
 				<div class="panel-body">
 					<form action="" method="POST" class="form">
 						{% csrf_token %}
+<!-- 						<div class="form-group">
```

Comment:
I removed the commented markups for this one. This page will change a lot for the next iteration.